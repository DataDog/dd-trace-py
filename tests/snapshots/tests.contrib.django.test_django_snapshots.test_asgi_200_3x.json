[[
  {
    "name": "django.request",
    "service": "django",
    "resource": "GET ^$",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "web",
    "meta": {
      "_dd.base_service": "",
      "_dd.p.dm": "-0",
      "asgi.version": "3.0",
      "component": "django",
      "django.request.class": "django.core.handlers.asgi.ASGIRequest",
      "django.response.class": "django.http.response.HttpResponse",
      "django.user.is_authenticated": "False",
      "django.view": "tests.contrib.django.views.index",
      "http.method": "GET",
      "http.route": "^$",
      "http.status_code": "200",
      "http.url": "http://localhost:8000/",
      "http.useragent": "python-requests/2.28.2",
      "http.version": "1.1",
      "language": "python",
      "runtime-id": "17b66fccf4474cc19fee336db9ed9836",
      "span.kind": "server"
    },
    "metrics": {
      "_dd.measured": 1,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "process_id": 20314
    },
    "duration": 3046833,
    "start": 1692647429697005426
  },
     {
       "name": "django.middleware",
       "service": "django",
       "resource": "django.contrib.sessions.middleware.SessionMiddleware.__call__",
       "trace_id": 0,
       "span_id": 2,
       "parent_id": 1,
       "meta": {
         "_dd.base_service": "",
         "component": "django"
       },
       "duration": 2085000,
       "start": 1692647429697536926
     },
        {
          "name": "django.middleware",
          "service": "django",
          "resource": "django.contrib.sessions.middleware.SessionMiddleware.process_request",
          "trace_id": 0,
          "span_id": 3,
          "parent_id": 2,
          "meta": {
            "_dd.base_service": "",
            "component": "django"
          },
          "duration": 43375,
          "start": 1692647429697571593
        },
        {
          "name": "django.middleware",
          "service": "django",
          "resource": "django.middleware.common.CommonMiddleware.__call__",
          "trace_id": 0,
          "span_id": 4,
          "parent_id": 2,
          "meta": {
            "_dd.base_service": "",
            "component": "django"
          },
          "duration": 1967458,
          "start": 1692647429697629593
        },
           {
             "name": "django.middleware",
             "service": "django",
             "resource": "django.middleware.common.CommonMiddleware.process_request",
             "trace_id": 0,
             "span_id": 6,
             "parent_id": 4,
             "meta": {
               "_dd.base_service": "",
               "component": "django"
             },
             "duration": 22667,
             "start": 1692647429697653301
           },
           {
             "name": "django.middleware",
             "service": "django",
             "resource": "django.middleware.csrf.CsrfViewMiddleware.__call__",
             "trace_id": 0,
             "span_id": 7,
             "parent_id": 4,
             "meta": {
               "_dd.base_service": "",
               "component": "django"
             },
             "duration": 1885459,
             "start": 1692647429697686134
           },
              {
                "name": "django.middleware",
                "service": "django",
                "resource": "django.middleware.csrf.CsrfViewMiddleware.process_request",
                "trace_id": 0,
                "span_id": 9,
                "parent_id": 7,
                "meta": {
                  "_dd.base_service": "",
                  "component": "django"
                },
                "duration": 11375,
                "start": 1692647429697700426
              },
              {
                "name": "django.middleware",
                "service": "django",
                "resource": "django.contrib.auth.middleware.AuthenticationMiddleware.__call__",
                "trace_id": 0,
                "span_id": 10,
                "parent_id": 7,
                "meta": {
                  "_dd.base_service": "",
                  "component": "django"
                },
                "duration": 1828625,
                "start": 1692647429697720676
              },
                 {
                   "name": "django.middleware",
                   "service": "django",
                   "resource": "django.contrib.auth.middleware.AuthenticationMiddleware.process_request",
                   "trace_id": 0,
                   "span_id": 12,
                   "parent_id": 10,
                   "meta": {
                     "_dd.base_service": "",
                     "component": "django"
                   },
                   "duration": 12750,
                   "start": 1692647429697743426
                 },
                 {
                   "name": "django.middleware",
                   "service": "django",
                   "resource": "django.contrib.messages.middleware.MessageMiddleware.__call__",
                   "trace_id": 0,
                   "span_id": 13,
                   "parent_id": 10,
                   "meta": {
                     "_dd.base_service": "",
                     "component": "django"
                   },
                   "duration": 1778917,
                   "start": 1692647429697765509
                 },
                    {
                      "name": "django.middleware",
                      "service": "django",
                      "resource": "django.contrib.messages.middleware.MessageMiddleware.process_request",
                      "trace_id": 0,
                      "span_id": 14,
                      "parent_id": 13,
                      "meta": {
                        "_dd.base_service": "",
                        "component": "django"
                      },
                      "duration": 29167,
                      "start": 1692647429697778384
                    },
                    {
                      "name": "django.middleware",
                      "service": "django",
                      "resource": "django.middleware.clickjacking.XFrameOptionsMiddleware.__call__",
                      "trace_id": 0,
                      "span_id": 15,
                      "parent_id": 13,
                      "meta": {
                        "_dd.base_service": "",
                        "component": "django"
                      },
                      "duration": 1703292,
                      "start": 1692647429697816301
                    },
                       {
                         "name": "django.middleware",
                         "service": "django",
                         "resource": "django.middleware.security.SecurityMiddleware.__call__",
                         "trace_id": 0,
                         "span_id": 17,
                         "parent_id": 15,
                         "meta": {
                           "_dd.base_service": "",
                           "component": "django"
                         },
                         "duration": 1661833,
                         "start": 1692647429697830468
                       },
                          {
                            "name": "django.middleware",
                            "service": "django",
                            "resource": "django.middleware.security.SecurityMiddleware.process_request",
                            "trace_id": 0,
                            "span_id": 19,
                            "parent_id": 17,
                            "meta": {
                              "_dd.base_service": "",
                              "component": "django"
                            },
                            "duration": 19584,
                            "start": 1692647429697843509
                          },
                          {
                            "name": "django.middleware",
                            "service": "django",
                            "resource": "tests.contrib.django.middleware.ClsMiddleware.__call__",
                            "trace_id": 0,
                            "span_id": 20,
                            "parent_id": 17,
                            "meta": {
                              "_dd.base_service": "",
                              "component": "django"
                            },
                            "duration": 1578542,
                            "start": 1692647429697872551
                          },
                             {
                               "name": "django.middleware",
                               "service": "django",
                               "resource": "tests.contrib.django.middleware.fn_middleware",
                               "trace_id": 0,
                               "span_id": 22,
                               "parent_id": 20,
                               "meta": {
                                 "_dd.base_service": "",
                                 "component": "django"
                               },
                               "duration": 1558500,
                               "start": 1692647429697886759
                             },
                                {
                                  "name": "django.middleware",
                                  "service": "django",
                                  "resource": "tests.contrib.django.middleware.EverythingMiddleware.__call__",
                                  "trace_id": 0,
                                  "span_id": 23,
                                  "parent_id": 22,
                                  "meta": {
                                    "_dd.base_service": "",
                                    "component": "django"
                                  },
                                  "duration": 1534875,
                                  "start": 1692647429697899884
                                },
                                   {
                                     "name": "django.middleware",
                                     "service": "django",
                                     "resource": "django.middleware.csrf.CsrfViewMiddleware.process_view",
                                     "trace_id": 0,
                                     "span_id": 24,
                                     "parent_id": 23,
                                     "meta": {
                                       "_dd.base_service": "",
                                       "component": "django"
                                     },
                                     "duration": 26583,
                                     "start": 1692647429698779551
                                   },
                                   {
                                     "name": "django.middleware",
                                     "service": "django",
                                     "resource": "tests.contrib.django.middleware.EverythingMiddleware.process_view",
                                     "trace_id": 0,
                                     "span_id": 25,
                                     "parent_id": 23,
                                     "meta": {
                                       "_dd.base_service": "",
                                       "component": "django"
                                     },
                                     "duration": 19667,
                                     "start": 1692647429698970676
                                   },
                                   {
                                     "name": "django.view",
                                     "service": "django",
                                     "resource": "tests.contrib.django.views.index",
                                     "trace_id": 0,
                                     "span_id": 26,
                                     "parent_id": 23,
                                     "meta": {
                                       "_dd.base_service": "",
                                       "component": "django"
                                     },
                                     "duration": 41708,
                                     "start": 1692647429699156968
                                   },
                          {
                            "name": "django.middleware",
                            "service": "django",
                            "resource": "django.middleware.security.SecurityMiddleware.process_response",
                            "trace_id": 0,
                            "span_id": 21,
                            "parent_id": 17,
                            "meta": {
                              "_dd.base_service": "",
                              "component": "django"
                            },
                            "duration": 22667,
                            "start": 1692647429699463551
                          },
                       {
                         "name": "django.middleware",
                         "service": "django",
                         "resource": "django.middleware.clickjacking.XFrameOptionsMiddleware.process_response",
                         "trace_id": 0,
                         "span_id": 18,
                         "parent_id": 15,
                         "meta": {
                           "_dd.base_service": "",
                           "component": "django"
                         },
                         "duration": 13416,
                         "start": 1692647429699500718
                       },
                    {
                      "name": "django.middleware",
                      "service": "django",
                      "resource": "django.contrib.messages.middleware.MessageMiddleware.process_response",
                      "trace_id": 0,
                      "span_id": 16,
                      "parent_id": 13,
                      "meta": {
                        "_dd.base_service": "",
                        "component": "django"
                      },
                      "duration": 11583,
                      "start": 1692647429699527468
                    },
              {
                "name": "django.middleware",
                "service": "django",
                "resource": "django.middleware.csrf.CsrfViewMiddleware.process_response",
                "trace_id": 0,
                "span_id": 11,
                "parent_id": 7,
                "meta": {
                  "_dd.base_service": "",
                  "component": "django"
                },
                "duration": 9208,
                "start": 1692647429699557051
              },
           {
             "name": "django.middleware",
             "service": "django",
             "resource": "django.middleware.common.CommonMiddleware.process_response",
             "trace_id": 0,
             "span_id": 8,
             "parent_id": 4,
             "meta": {
               "_dd.base_service": "",
               "component": "django"
             },
             "duration": 12500,
             "start": 1692647429699579134
           },
        {
          "name": "django.middleware",
          "service": "django",
          "resource": "django.contrib.sessions.middleware.SessionMiddleware.process_response",
          "trace_id": 0,
          "span_id": 5,
          "parent_id": 2,
          "meta": {
            "_dd.base_service": "",
            "component": "django"
          },
          "duration": 11917,
          "start": 1692647429699604801
        }]]
