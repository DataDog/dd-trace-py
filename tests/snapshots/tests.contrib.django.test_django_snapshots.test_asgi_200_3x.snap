[[{"name" "django.request"
   "service" "django"
   "resource" "GET ^$"
   "type" "web"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1628305913986026822
   "duration" 6168995
   "meta" {"runtime-id" "6cb9a1572abc45eea6f6f1c5c2cb749b"
           "http.route" "^$"
           "http.url" "http://localhost:8000/"
           "django.user.is_authenticated" "False"
           "http.status_code" "200"
           "http.method" "GET"
           "http.version" "1.1"
           "django.response.class" "django.http.response.HttpResponse"
           "django.request.class" "django.core.handlers.asgi.ASGIRequest"
           "django.view" "tests.contrib.django.views.index"
           "asgi.version" "3.0"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.measured" 1
              "_sampling_priority_v1" 1
              "system.pid" 560
              "_dd.tracer_kr" 1.0}}
      {"name" "django.middleware"
       "service" "django"
       "resource" "django.contrib.sessions.middleware.SessionMiddleware.__call__"
       "error" 0
       "span_id" 1
       "trace_id" 0
       "parent_id" 0
       "start" 1628305913986769436
       "duration" 3109724}
          {"name" "django.middleware"
           "service" "django"
           "resource" "django.contrib.sessions.middleware.SessionMiddleware.process_request"
           "error" 0
           "span_id" 2
           "trace_id" 0
           "parent_id" 1
           "start" 1628305913986797995
           "duration" 48155}
          {"name" "django.middleware"
           "service" "django"
           "resource" "django.middleware.common.CommonMiddleware.__call__"
           "error" 0
           "span_id" 3
           "trace_id" 0
           "parent_id" 1
           "start" 1628305913986868073
           "duration" 2960689}
              {"name" "django.middleware"
               "service" "django"
               "resource" "django.middleware.common.CommonMiddleware.process_request"
               "error" 0
               "span_id" 5
               "trace_id" 0
               "parent_id" 3
               "start" 1628305913986890210
               "duration" 38152}
              {"name" "django.middleware"
               "service" "django"
               "resource" "django.middleware.csrf.CsrfViewMiddleware.__call__"
               "error" 0
               "span_id" 6
               "trace_id" 0
               "parent_id" 3
               "start" 1628305913986992983
               "duration" 2743409}
                  {"name" "django.middleware"
                   "service" "django"
                   "resource" "django.middleware.csrf.CsrfViewMiddleware.process_request"
                   "error" 0
                   "span_id" 8
                   "trace_id" 0
                   "parent_id" 6
                   "start" 1628305913987017660
                   "duration" 11354}
                  {"name" "django.middleware"
                   "service" "django"
                   "resource" "django.contrib.auth.middleware.AuthenticationMiddleware.__call__"
                   "error" 0
                   "span_id" 9
                   "trace_id" 0
                   "parent_id" 6
                   "start" 1628305913987046976
                   "duration" 2654563}
                      {"name" "django.middleware"
                       "service" "django"
                       "resource" "django.contrib.auth.middleware.AuthenticationMiddleware.process_request"
                       "error" 0
                       "span_id" 11
                       "trace_id" 0
                       "parent_id" 9
                       "start" 1628305913987143004
                       "duration" 16685}
                      {"name" "django.middleware"
                       "service" "django"
                       "resource" "django.contrib.messages.middleware.MessageMiddleware.__call__"
                       "error" 0
                       "span_id" 12
                       "trace_id" 0
                       "parent_id" 9
                       "start" 1628305913987178853
                       "duration" 2514941}
                          {"name" "django.middleware"
                           "service" "django"
                           "resource" "django.contrib.messages.middleware.MessageMiddleware.process_request"
                           "error" 0
                           "span_id" 13
                           "trace_id" 0
                           "parent_id" 12
                           "start" 1628305913987199658
                           "duration" 108460}
                          {"name" "django.middleware"
                           "service" "django"
                           "resource" "django.middleware.clickjacking.XFrameOptionsMiddleware.__call__"
                           "error" 0
                           "span_id" 14
                           "trace_id" 0
                           "parent_id" 12
                           "start" 1628305913987328471
                           "duration" 2327559}
                              {"name" "django.middleware"
                               "service" "django"
                               "resource" "django.middleware.security.SecurityMiddleware.__call__"
                               "error" 0
                               "span_id" 16
                               "trace_id" 0
                               "parent_id" 14
                               "start" 1628305913987393625
                               "duration" 2173857}
                                  {"name" "django.middleware"
                                   "service" "django"
                                   "resource" "django.middleware.security.SecurityMiddleware.process_request"
                                   "error" 0
                                   "span_id" 18
                                   "trace_id" 0
                                   "parent_id" 16
                                   "start" 1628305913987416560
                                   "duration" 10178}
                                  {"name" "django.middleware"
                                   "service" "django"
                                   "resource" "tests.contrib.django.middleware.ClsMiddleware.__call__"
                                   "error" 0
                                   "span_id" 19
                                   "trace_id" 0
                                   "parent_id" 16
                                   "start" 1628305913987443978
                                   "duration" 2065900}
                                      {"name" "django.middleware"
                                       "service" "django"
                                       "resource" "tests.contrib.django.middleware.EverythingMiddleware"
                                       "error" 0
                                       "span_id" 21
                                       "trace_id" 0
                                       "parent_id" 19
                                       "start" 1628305913987491956
                                       "duration" 2009318}
                                          {"name" "django.middleware"
                                           "service" "django"
                                           "resource" "tests.contrib.django.middleware.EverythingMiddleware.__call__"
                                           "error" 0
                                           "span_id" 22
                                           "trace_id" 0
                                           "parent_id" 21
                                           "start" 1628305913987512601
                                           "duration" 1975442}
                                              {"name" "django.middleware"
                                               "service" "django"
                                               "resource" "django.middleware.csrf.CsrfViewMiddleware.process_view"
                                               "error" 0
                                               "span_id" 23
                                               "trace_id" 0
                                               "parent_id" 22
                                               "start" 1628305913988085047
                                               "duration" 67997}
                                              {"name" "django.middleware"
                                               "service" "django"
                                               "resource" "tests.contrib.django.middleware.EverythingMiddleware.process_view"
                                               "error" 0
                                               "span_id" 24
                                               "trace_id" 0
                                               "parent_id" 22
                                               "start" 1628305913988411477
                                               "duration" 14573}
                                              {"name" "django.view"
                                               "service" "django"
                                               "resource" "tests.contrib.django.views.index"
                                               "error" 0
                                               "span_id" 25
                                               "trace_id" 0
                                               "parent_id" 22
                                               "start" 1628305913988982482
                                               "duration" 74335}
                                  {"name" "django.middleware"
                                   "service" "django"
                                   "resource" "django.middleware.security.SecurityMiddleware.process_response"
                                   "error" 0
                                   "span_id" 20
                                   "trace_id" 0
                                   "parent_id" 16
                                   "start" 1628305913989532322
                                   "duration" 26989}
                              {"name" "django.middleware"
                               "service" "django"
                               "resource" "django.middleware.clickjacking.XFrameOptionsMiddleware.process_response"
                               "error" 0
                               "span_id" 17
                               "trace_id" 0
                               "parent_id" 14
                               "start" 1628305913989629467
                               "duration" 18836}
                          {"name" "django.middleware"
                           "service" "django"
                           "resource" "django.contrib.messages.middleware.MessageMiddleware.process_response"
                           "error" 0
                           "span_id" 15
                           "trace_id" 0
                           "parent_id" 12
                           "start" 1628305913989674535
                           "duration" 12103}
                  {"name" "django.middleware"
                   "service" "django"
                   "resource" "django.middleware.csrf.CsrfViewMiddleware.process_response"
                   "error" 0
                   "span_id" 10
                   "trace_id" 0
                   "parent_id" 6
                   "start" 1628305913989718930
                   "duration" 10376}
              {"name" "django.middleware"
               "service" "django"
               "resource" "django.middleware.common.CommonMiddleware.process_response"
               "error" 0
               "span_id" 7
               "trace_id" 0
               "parent_id" 3
               "start" 1628305913989753532
               "duration" 64959}
          {"name" "django.middleware"
           "service" "django"
           "resource" "django.contrib.sessions.middleware.SessionMiddleware.process_response"
           "error" 0
           "span_id" 4
           "trace_id" 0
           "parent_id" 1
           "start" 1628305913989849790
           "duration" 21427}]]
